go/sdk/v3/models.go

```diff
@@ -1067,23 +1067,23 @@ type ImportedProject struct {
-  IconUrl                           *string                                `json:"icon_url,omitempty"`                             // URL to an icon for the integration.
-  UsesOauth                         *bool                                  `json:"uses_oauth,omitempty"`                           // Whether the integration uses oauth.
-  RequiredFields                    *[]IntegrationRequiredField            `json:"required_fields,omitempty"`                      // A list of descriptions of required fields that this integration is compatible with. If there are multiple entries in this list, the integration requires more than one field. If unspecified, no fields will be required.
-  DelegateOauth                     *bool                                  `json:"delegate_oauth,omitempty"`                       // Whether the integration uses delegate oauth, which allows federation between an integration installation scope specific entity (like org, group, and team, etc.) and Looker.
-  InstalledDelegateOauthTargets     *[]int64                               `json:"installed_delegate_oauth_targets,omitempty"`     // Whether the integration is available to users.
+  Can                               *map[string]bool                     `json:"can,omitempty"`                                  // Operations the current user is able to perform on this object
+  Id                                *string                              `json:"id,omitempty"`                                   // ID of the integration.
+  IntegrationHubId                  *int64                               `json:"integration_hub_id,omitempty"`                   // ID of the integration hub.
+  Label                             *string                              `json:"label,omitempty"`                                // Label for the integration.
+  Description                       *string                              `json:"description,omitempty"`                          // Description of the integration.
+  Enabled                           *bool                                `json:"enabled,omitempty"`                              // Whether the integration is available to users.
+  Params                            *[]IntegrationParam                  `json:"params,omitempty"`                               // Array of params for the integration.
+  SupportedFormats                  *[]SupportedFormats                  `json:"supported_formats,omitempty"`                    // A list of data formats the integration supports. If unspecified, the default is all data formats. Valid values are: "txt", "csv", "inline_json", "json", "json_label", "json_detail", "json_detail_lite_stream", "xlsx", "html", "wysiwyg_pdf", "assembled_pdf", "wysiwyg_png", "csv_zip".
```

Comment:
@kppk should this be `[]SupportedFormatType` instead of `[]SupportedFormats`? if so, then there's some misconfiguration between packages/sdk-codegen/src/sdkModels.ts and packages/sdk-codegen/src/go.gen.ts